internal/services/applications/parse/identifier_uri.go

```diff
@@ -74,3 +74,17 @@ func (id IdentifierUriId) Segments() []resourceids.Segment {
 func (id IdentifierUriId) String() string {
 	return fmt.Sprintf("Application IdentifierUri (Application ID: %q, IdentifierUri ID: %q)", id.ApplicationId, id.IdentifierUri)
 }
+
+func (id IdentifierUriId) FromParseResult(input resourceids.ParseResult) error {
```

Comment:
This'll need to be a pointer:

```suggestion
func (id *IdentifierUriId) FromParseResult(input resourceids.ParseResult) error {
```